tests/main/snapctl-disable-services-install-hook/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Check that `snapctl stop --disable` actually stops services on install
+
+execute: |
+    # shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local snapctl-disable-services-install
```

Comment:
Did you mean to say `...-hook` here? Specifically `snapctl-disable-services-install-hook`?

Btw, the general naming we are trying to go after is `test-snapd-foo`. This is how we name internal snaps that only exists for testing.

Reply:
Yes this is another typo... Also I wanted to follow the naming convention, but my initial attempt at:
`test-snapctl-disable-services-install-hook` is over the 40 character limit for snap names, so I removed the `test-`. Is there a more concise name I'm not seeing here?